contracts/0.4.24/Lido.sol

```diff
@@ -597,13 +597,13 @@ contract Lido is ILido, IsContract, Pausable, AragonApp {
         uint256 totalPooledEther = _getTotalPooledEther();
         uint256 shares2mint = (
             feeInEther
-            .mul(stEth.getTotalShares())
+            .mul(token.getTotalShares())
             .div(totalPooledEther.sub(feeInEther))
         );
 
         // Mint the calculated amount of shares to this contract address. This will reduce the
         // balances of the holders, as if the fee was taken in parts from each of them.
-        uint256 totalShares = stEth.mintShares(address(this), shares2mint);
+        uint256 totalShares = token.mintShares(address(this), shares2mint);
```

Comment:
I definitely support this. 
* Lido has a single tightly-coupled token, no need to distinguish 
* Already had a confusion stEth vs StEth